tomls/omnibus-base-mainnet-andromeda/perps/btc.toml

```diff
@@ -0,0 +1,54 @@
+[setting.pythBtcFeedId]
+defaultValue = "0xe62df6c8b4a85fe1a67db44dc12de5db330f7ac66b72dc658afedf0f4a415b43" # Using mainnet feed IDs
+
+# Perps Configuration
+[setting.btcPerpsMarketId]
+defaultValue = "200"
+
+[setting.perpsBtcSkewScale]
+defaultValue = "100000"
+
+[setting.perpsBtcMaxFundingVelocity]
+defaultValue = "9"
+
+[setting.perpsBtcMakerFeeRatio]
+defaultValue = "0.0002"
+
+[setting.perpsBtcTakerFeeRatio]
+defaultValue = "0.0005"
+
+[setting.perpsBtcMaxMarketSize]
+defaultValue = "0.25" #approx $10k
+
+[setting.perpsBtcInitialMarginRatio]
+defaultValue = "13.35"
+
+[setting.perpsBtcMaintenanceMarginScalar]
+defaultValue = "0.28"
+
+[setting.perpsBtcMinimumInitialMarginRatio]
+defaultValue = "0.02"
+
+[setting.perpsBtcFlagRewardRatioD18]
+defaultValue = "0.0003"
+
+[setting.perpsBtcMaxLiquidationLimitAccumulationMultiplier]
+defaultValue = "1.5"
+
+[setting.perpsBtcMaxSecondsInLiquidationWindow]
+defaultValue = "30"
+
+[setting.perpsBtcMinimumPositionMargin]
+defaultValue = "50"
+
+[setting.perpsBtcLockedOiRatio]
+defaultValue = "0.5"
+
+[setting.perpsBtcSynthMaxCollateralAmount]
+defaultValue = "<%= MaxUint256 %>"
+
+[setting.perpsBtcMaxLiquidationPd]
+defaultValue = "0.0005"
+
+[setting.perpsBtcEndorsedLiquidator]
+defaultValue = "0x11233749514Ab8d00C0A5873DF7428b3db70030f"
```

Comment:
Same comment here too: this value is the same as testnet. If it should be - nevermind then